_data_weighting.tex

```diff
@@ -210,7 +210,7 @@ \subsection{Data Weighting}
 
 If the reported $\theta/(1+\theta)$ ratio is close to 1.0, that indicates that the model is trying to tune the sample size as high as possible. In this case, the $ln(\theta)$ parameters should be fixed at a high value, like the upper bound of 20, which will result in 100\% weight being applied to the input sample sizes. An alternative would be to manually change the input sample sizes to a higher value so that the estimated weighting will be less than 100%.
 
-Note that a constant of integration was added to the diricihlet-multinomila likelihood equation in SS3 v.3.30.17. This will change the likelihood value, but parameter estimates and expected values should remain the same as in previous versons of SS3.
+Note that a constant of integration was added to the Dirichlet-Multinomial likelihood equation in SS3 v.3.30.17. This will change the likelihood value, but parameter estimates and expected values should remain the same as in previous versions of SS3.
```

Comment:
Good catch on the misspelled "multinomila".
Regarding capitalization, I recently realized that the Thorson et al. paper which describes this new feature as implemented in SS3 only capitalizes the "D": "Dirichlet-multinomial" (see https://doi.org/10.1016/j.fishres.2016.06.005). That's not to say we have to use that approach, and the abbreviation should be "DM" or "D-M" regardless.

Reply:
I would defer to whatever @chantelwetzel-noaa thinks regarding capitalization.

Reply:
Dirichlet-multinomial is technically correct because Dirichlet is a name of a person. See [wikipedia](https://en.wikipedia.org/wiki/Dirichlet_distribution).